BioinformaticsProject

```diff
@@ -0,0 +1,64 @@
+# Script prepared by Isaiah Murrell-Thomas and Bethany Oceguera 
+# Assume that you are in the bioinformaticsProject directory. This is where you MUST run the script :)
+# Here, you have to assume that the script and files used are in the bioinformaticsProject directory 
+# Usage : bash BioinformaticsProject [muscle tool path] [hmmbuild tool path] [hmmsearch path]
+
+# Example Code to run
+# bash BioinformaticsProject ~/Private/Biocomputing2022/tools/muscle ~/Private/Biocomputing2022/tools/hmmbuild ~/Private/Biocomputing2022/tools/hmmsearch
+
+# CODE START
+
+# Create one file that appends all McrA gene files 
+for A in ./ref_sequences/mcrA*.fasta
+do
+cat $A >> allspecies.fasta
+done
+
```

Comment:
You actually don't need the for loop for this
cat $1/mcrAgene_* > $1/ref_mcrAgenes.fasta
+1